### DISCUSSION TITLE
error while publishing package to gitlab · package manifest is not coherent how

### DISCUSSION THREAD

**Discussion by nitesh-habilelabs** [teambit/bit#9049] [Help]
Hi, 
we are using self hosted gitlab and self hosted bit and using bit tag, we are publishing those bit components to package registry of gitlab. it was working fine earlier. but recently when i am publishing (bit tag) package on gitlab, it is giving me error that “error publishing · package manifest is not coherent”.

i tried looking for this issue on various sites and GPTs but failed to resolve this. how to fix that? any idea or help is appreciated.

Thank you.

**Comment by itaymendel** [comment]
hi, there's not much of information on the issue. can you please provide more information?

**Reply by nitesh-habilelabs** [reply]
> hi, there's not much of information on the issue. can you please provide more information?

Hi, this issue is resolved, as we found out later that this issue occured from gitlab side. Thank you for your awesome work.
